com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/util/PropertiesUtil.java

```diff
@@ -143,10 +144,10 @@ public static String transformResourcePropertyValue(String value) {
     }
 
     public static void createResourcePropertyTasks(Project project, IResource resource, IResource input) throws CompileExceptionError {
-        //Textures
-        Class<? extends Builder<?>> klass = TextureBuilder.class;
-        if (project.getBuilderFromExtension(resource) == klass) {
-            Task<?> embedTask = project.createTask(resource, klass);
+        // Textures and cubemaps
```

Comment:
We still can't build all the other resource types here, because we will get building conflicts in this case.